aptos-move/vm-genesis/src/lib.rs

```diff
@@ -717,7 +742,7 @@ fn create_and_initialize_validators_with_commission(
     let validators_bytes = bcs::to_bytes(validators).expect("Validators can be serialized");
     let mut serialized_values = serialize_values(&vec![
         MoveValue::Signer(CORE_CODE_ADDRESS),
-        MoveValue::Bool(true),
+        MoveValue::Bool(false),
```

Comment:
I think we would not be calling this function of `genesis` since we always use pool initialization function to initialize the validators, isn't it?

Reply:
Yep, this is a residue of an old approach. I'll undo this change.

Reply:
Addressed in 44e74f3e81e55dbd58cac2834abaf383ba2eed1a